Navaids/VOR_Non-UK.txt

```diff
@@ -10,41 +10,41 @@ RTM 110.400 N051.58.25.310 E004.28.51.490 ; Rotterdam
 SPL 108.400 N052.19.55.720 E004.44.59.580 ; Schiphol
 SPY 113.300 N052.32.25.010 E004.51.13.610 ; Spijkerboor (DME)
 
-;Belgium, verified 2012/01
+;Belgium, verified 2308
 AFI 114.900 N050.54.28.000 E004.08.20.000 ; Affligem
 ANT 113.500 N051.11.26.000 E004.28.21.000 ; Antwerpen
-BBE 116.000 N050.45.23.000 E004.46.05.000 ; Beauvechain (TACAN Ch107X)
-BBL 108.600 N051.10.13.000 E005.27.59.000 ; Kleine-Brogel (TACAN Ch33X)
-BFS 110.500 N050.14.29.000 E004.39.11.000 ; Florennes (TACAN Ch52X)
+BBE 116.000 N050.45.25.000 E004.46.07.000 ; Beauvechain (TACAN Ch107X)
+BBL 108.600 N051.10.03.000 E005.27.51.000 ; Kleine-Brogel (TACAN Ch33X)
+BFS 110.500 N050.14.29.000 E004.39.12.000 ; Florennes (TACAN Ch52X)
 BUB 114.600 N050.54.08.000 E004.32.17.000 ; Brussels
 BUN 110.600 N051.07.07.000 E004.50.32.000 ; Bruno
-CIV 113.200 N050.34.26.000 E003.49.58.000 ; Chi�vres
+CIV 113.200 N050.34.26.000 E003.49.58.000 ; Chièvres
 COA 110.050 N051.20.53.000 E003.21.19.000 ; Costa
 DIK 114.400 N049.51.41.000 E006.07.47.000 ; Diekirch
 FLO 112.050 N050.52.36.000 E005.08.04.000 ; Flora
 GSY 115.700 N050.27.14.000 E004.26.29.000 ; Gosly
 HUL 117.550 N050.44.58.000 E004.38.30.000 ; Huldenberg
 KOK 114.500 N051.05.41.000 E002.39.06.000 ; Koksy
-LGE 115.450 N050.39.14.000 E005.28.14.000 ; Li�ge
+LGE 115.450 N050.39.14.000 E005.28.14.000 ; Liège
 LNO 112.800 N050.35.09.000 E005.42.37.000 ; Olno
 LUX 112.250 N049.38.22.000 E006.14.50.000 ; Luxembourg
 NIK 117.400 N051.09.54.000 E004.11.02.000 ; Nicky
 SPI 113.100 N050.30.53.000 E005.37.25.000 ; Sprimont
 
-;France
+;France, verified AIRAC 2308
```

Comment:
Quite a few french ones (above lattitude 45N which I think was the cut off) seem to be missing, not sure if this is intentional?

Reply:
Won't lie I just removed the ones below N045 instead of checking all against ENR4.4, I'll add in the ones that are above, I didn't really think about that to be honest.